modules/admission-webhook-create.adoc

```diff
@@ -0,0 +1,328 @@
+  subjects:
+  - kind: ServiceAccount
+    namespace: <namespace_name>
+    name: server
+
+- apiVersion: rbac.authorization.k8s.io/v1  <2>
+  kind: ClusterRole
+  metadata:
+    annotations:
+    name: system:openshift:online:<webhook-name>-server
+  rules:
+  - apiGroups:
+    - online.openshift.io
+    resources:
+    - <resource_name>
+    verbs:
+    - get
+    - list
+    - watch
+
+- apiVersion: rbac.authorization.k8s.io/v1  <3>
+  kind: ClusterRole
+  metadata:
+    name: system:openshift:online:<webhook_name>-requester
+  rules:
+  - apiGroups:
+    - admission.online.openshift.io
+    resources:
+    - <resource_name>
+    verbs:
+    - create
+
+- apiVersion: rbac.authorization.k8s.io/v1  <4>
+  kind: ClusterRoleBinding
+  metadata:
+    name: namespace-reservation-server-<namespace_name>
+  roleRef:
+    kind: ClusterRole
+    apiGroup: rbac.authorization.k8s.io
+    name: system:openshift:online:<webhook_name>-server
+  subjects:
+  - kind: ServiceAccount
+    namespace: <namespace_name>
+    name: server
+
+- apiVersion: rbac.authorization.k8s.io/v1  <5>
+  kind: RoleBinding
+  metadata:
+    namespace: kube-system
+    name: extension-server-authentication-reader-<namespace_name>
+  roleRef:
+    kind: Role
+    apiGroup: rbac.authorization.k8s.io
+    name: extension-apiserver-authentication-reader
+  subjects:
+  - kind: ServiceAccount
+    namespace: <namespace_name>
+    name: server
+
+- apiVersion: rbac.authorization.k8s.io/v1  <6>
+  kind: ClusterRole
+  metadata:
+    name: <aggregated_apiserver_cluster_role>
+  rules:
+  - apiGroups:
+    - admissionregistration.k8s.io
+    resources:
+    - validatingwebhookconfigurations
+    - mutatingwebhookconfigurations
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - ""
+    resources:
+    - namespaces
+    verbs:
+    - get
+    - list
+    - watch
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: <aggregated_apiserver_cluster_role>
+  roleRef:
+    kind: ClusterRole
+    apiGroup: rbac.authorization.k8s.io
+    name: <aggregated_apiserver_cluster_role>
+  subjects:
+  - kind: ServiceAccount
+    namespace: <namespace_name>
+    name: server
+----
+<1> Delegates authentication and authorization to the webhook server API.
+<2> Allows the admission webhook server to access cluster resource.
+<3> Enables the aggregated API server to create admission reviews.
+<4> Also enables the admission server to access cluster resource.
+<5> Role binding to read the configuration for terminating authentication.
+<6> Default cluster role and cluster role bindings for an aggregated API server.
+
+. Apply those RBAC rules to the cluster:
++
+----
+$ oc auth reconcile -f <YAML_filename>
+----
+
+. Create a new project to run the DaemonSet:
++
+----
+$ oc create namespace <namespace_name>
+----
+
+. Define a YAML configuration that is used to deploy a webhook as a DaemonSet server in a project. Restrict it to master nodes only:
```

Comment:
It's unclear to me where/how you restrict it to master nodes only.

Reply:
@deads2k in https://github.com/openshift/kubernetes-namespace-reservation/blob/master/hack/install-kube.sh you mentioned that preferably the DaemonSet should be deployed on master nodes only. Can you please advise as to how to best achieve that? Should we introduce a nodeselctor item in the DaemonSet definition?

Reply:
Following a conversation with @vikram-redhat just now, we have decided to remove the reference to master nodes here, because in David's comment within the script, this is cited as a preference, not a requirement.